src/Parser/EnhancementShaman/Modules/ShamanCore/DamageDone.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import Icon from 'common/Icon';
+import { formatNumber, formatPercentage } from 'common/format';
+
+import CoreDamageDone from 'Parser/Core/Modules/DamageDone';
+
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class DamageDone extends CoreDamageDone {
+  suggestions(when) {
+    const nonDpsTimePercentage = this.owner.modules.alwaysBeCasting.totalTimeWasted / this.owner.fightDuration;
+    when(nonDpsTimePercentage).isGreaterThan(0.3)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(`Your non DPS time can be improved. Try to cast damaging spells more regularly`)
+          .icon("petbattle_health")
+          .actual(`${Math.round(nonDpsTimePercentage * 100)}% non DPS time`)
```

Comment:
This should use the `actual` value instead, so you don't have to worry about keeping both values in sync when the threshold changes.